### PR TITLE
Merge Alert Component

### PR DESCRIPTION
**Pull request by Hamdrive**
Created 3 types of alerts:
1. Basic Alert w/ text and color
2. Alert with icons
3. Alert with hyperlink text

All 3 types of alerts also have hover states applied to them

AlertsGIF

### CODE REVIEW

File: components/alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alert • PowertrainUI</title>
+  </head>
+  <body>
+    <main class="wrapper">
+      <nav class="wrapper__nav-top">
```
Comment (junaidshaikh-js) [inline components/alert/alert.html]: navigation should be inside the `header` tag not `main` 

Also, don't use `main` as a wrapper, you can use `div` for general-purpose styling.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+/* Only for Visualization of Example */
+
+
+/* Alert Component CSS */
+.alert{
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  font-weight: 600;
+  height: 50px;
```
Comment (aryan-13) [inline components/alert/alert.css]: Using rem is better for responsiveness

File: components/alert/alert.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            <h2 class="nav__title">PowertrainUI</h2>
+            <img
+              class="nav__img"
+              src="./images/icon.png"
+              alt="PowertrainUI logo"
+            />
+          </a>
+        </div>
+        <ul class="nav__menu">
+          <li class="nav__menu-item"><a href="#">Get Started</a></li>
```
Comment (junaidshaikh-js) [inline components/alert/alert.html]: Looks like you are yet to add the links to your anchor tag. 

Add it first then merge the branch.

**Review by aryan-13 — COMMENTED** [review]
Everything else looks good!

File: components/alert/alert.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+          <li class="nav__menu-item"><a href="#">Docs</a></li>
+        </ul>
+      </nav>
+      <section class="main-content">
+        <aside></aside>
+        <article></article>
+      </section>
+      <footer class="wrapper__footer">
```
Comment (junaidshaikh-js) [inline components/alert/alert.html]: footer should have it's own space, it shouldn't be inside `main` 

General structure is like 

```html
<body>
  <header></header>
  <main></main>
  <footer></footer>
</body>

```

File: components/alert/alert.css [hunk 5]
```diff
@@ -14,38 +28,66 @@
...
+  color: var(--alert-warning-icon);
 }
 
-.alert-warning{
-  background-color: #fcd34d;
-  color: #92400e;
+.alert--warning:hover{
+  cursor: pointer;
+  opacity: 0.85;
+  transition: 0.1s ease-in-out;
```
Comment (junaidshaikh-js) [inline components/alert/alert.css]: it's good to define which property you are apply transition to 

```css
transition: background-color 0.1s linear;
```

File: components/alert/alert.css (hunk 2 above)
Comment (Hamdrive) [reply components/alert/alert.css]: Thanks, have implemented the change. Good catch!

File: components/alert/alert.html (hunk 1 above)
Comment (Hamdrive) [reply components/alert/alert.html]: Not focusing on docs html but valuable points noted for later!

File: components/alert/alert.html (hunk 3 above)
Comment (Hamdrive) [reply components/alert/alert.html]: Will add anchor tag links when working on doc html

File: components/alert/alert.html (hunk 4 above)
Comment (Hamdrive) [reply components/alert/alert.html]: Good catch, will revamp html structure when I work on it!

File: components/alert/alert.css (hunk 5 above)
Comment (Hamdrive) [reply components/alert/alert.css]: Did not know you can apply transition like this. Thanks for showing this!

**Pull request merged**